### PR TITLE
GitHub actions port 2

### PR DESCRIPTION
**Pull request by chick**
**Related issue**: 

**Type of change**: bug fix | new feature | other enhancement

**Impact**: rtl change | software change | unknown | other

**Release Notes**

### CODE REVIEW

File: .github/wip/create-hashes.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: pactice-actions
+
+on: [push]
+
+jobs:
+  create-hashes:
+    name: create-hashes
+    runs-on: ubuntu-latest
+    container:
+      image: ucbbar/chipyard-image:1.0.1
```
Comment (abejgonzalez) [inline .github/wip/create-hashes.yml:10]:
```suggestion
      image: ucbbar/chipyard-ci-image:9c650dea
```

File: .github/scripts/defaults.sh [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+# make parallelism
+CI_MAKE_NPROC=8
+# chosen based on a 24c system shared with 1 other project
+REMOTE_MAKE_NPROC=4
+
+# verilator version
+VERILATOR_VERSION=v4.034
+
+pwd
```
Comment (abejgonzalez) [inline .github/scripts/defaults.sh]:
```suggestion
pwd
git --version
```

File: .github/workflows/chipyard-ci-process.yml [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    name: cache-test
+    runs-on: ubuntu-latest
+    container:
+      image: ucbbar/chipyard-ci-image:9c650dea
+      options: --user root --entrypoint /bin/bash
+    env:
+      JVM_OPTS: -Xmx3200m # Customize the JVM maximum heap limit
+    steps:
+#      - name: fix permissions 1
+#        run: sudo chown -R riscvuser .
```
Comment (abejgonzalez) [inline .github/workflows/chipyard-ci-process.yml]:
```suggestion
      - name: Check git version
        run: git --version
```

**Comment by chick** [comment]
working on next pr now. closing this

**Pull request closed without merging**